include/boost/gil/detail/math.hpp

```diff
@@ -10,15 +11,29 @@
 
 #include <array>
 #include <boost/gil/extension/numeric/kernel.hpp>
+#include <vector>
```

Comment:
Please, [bundle the C++ standard headers together](https://github.com/boostorg/gil/wiki/Include-Directives-Order)